src/modules/fetchArticle.js

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+const fetchWrapper = (setSelectedArticle, setPreviewMessage, id) => {
+  let fetchSelectedArticle = async (id) => {
+    try {
+      let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+      headers = {
+        ...headers,
+        "Content-type": "application/json",
+        Accept: "application/json",
+      };
```

Comment:
I see this headers setter repeated. Again, perhaps it's better to create a wrapper around it and place it in a module or service?